ctapipe/image/muon/intensity_fitter.py

```diff
@@ -195,6 +196,28 @@ def image_prediction(
     )
 
 
+@vectorize([double(double, double, double)])
+def gaussian_cdf(x, mu, sig):
+    """
+    Function to compute values of a given gaussians
+    cumulative distribution function (cdf)
+
+    Parameters
+    ----------
+    x: float
+        point, at which the cdf should be evaluated
+    mu: float
+        gaussian mean
+    sig: float
+        gaussian standart deviation
+
+    Returns
+    -------
+    float: cdf-value at x
+    """
+    return 0.5 * (1 + erf((x - mu) / (np.sqrt(2 * sig ** 2))))
```

Comment:
sqrt is a pretty slow operation. sig is positive. So maybe we can gain something by defining a constant `sqrt2` and removing the sqrt here?